<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ApmAssistant</class>
 <widget class="QWidget" name="ApmAssistant">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>268</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QWebView" name="webView">
       <property name="url">
        <url>
         <string>about:blank</string>
        </url>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QToolButton" name="goBackButton">
         <property name="minimumSize">
          <size>
           <width>59</width>
           <height>23</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>23</height>
          </size>
         </property>
         <property name="text">
          <string>Go Back</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>188</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="reloadButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../../qgroundcontrol.qrc">
           <normaloff>:/files/images/actions/go-jump.svg</normaloff>:/files/images/actions/go-jump.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="nextButton">
         <property name="minimumSize">
          <size>
           <width>42</width>
           <height>23</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>23</height>
          </size>
         </property>
         <property name="text">
          <string>Next</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <zorder></zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../qgroundcontrol.qrc"/>
 </resources>
 <connections/>
</ui>
